chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/CertifiedProductManagerImpl.java

```diff
@@ -2173,8 +2173,8 @@ private void triggerDeveloperBan(final CertifiedProductSearchDetails updatedList
                     .usingJobData("developer", updatedListing.getDeveloper().getName())
                     .usingJobData("acb", updatedListing.getCertifyingBody().get("name").toString())
                     .usingJobData("changeDate", new Date().getTime())
-                    .usingJobData("firstName", Util.getCurrentUser().getFirstName())
-                    .usingJobData("lastName", Util.getCurrentUser().getLastName())
+                    .usingJobData("firstName", Util.getCurrentUser().getFullName())
+                    .usingJobData("lastName", Util.getCurrentUser().getFriendlyName())
```

Comment:
Does it matter that "firstName" and "lastName" are still the keys here? Probably should be full and friendly anyway?